src/eth-signing/helpers.ts

```diff
@@ -133,6 +133,28 @@ export function fixRawSignature(
   }
 }
 
+/**
+ * Return signature with rotated 'v' value.
+ * @param signature Hex stripped raw signature
+ * @returns a rotated version of the inputted signature.
+ */
```

Comment:
needs more context and possibly error-checking about expected behavior
I think for example it expects that the signature is not `0x` prepended?
What happens if it can't rotate the signature? etc